core/templates/dev/head/domain/learner_dashboard/LearnerDashboardIconsDirective.js

```diff
@@ -59,6 +59,20 @@ oppia.directive('learnerDashboardIcons', [
                   response.data.learner_dashboard_activity_ids));
             }
           );
+          var learnerPlaylistUrl = (
+            UrlInterpolationService.interpolateUrl(
+              '/learnerplaylistactivityhandler/<activityType>/<activityId>', {
+                activityType: $scope.getActivityType(),
+                activityId: $scope.getActivityId()
+              }));
+          $http.post(learnerPlaylistUrl, {})
+            .then(function(response) {
+              if (response.data.belongs_to_subscribed_activities) {
+                $scope.availableToAddByCreator = false;
```

Comment:
Maybe use 'shouldShowAddToPlaylistIcon' as it is never available to add by creator.

Reply:
ok